conclave-common/src/main/kotlin/com/r3/conclave/common/internal/EnclaveCallType.kt

```diff
@@ -0,0 +1,28 @@
+package com.r3.conclave.common.internal
+
+/**
+ * Enumeration of all possible host -> enclave calls
+ */
+enum class EnclaveCallType {
+    INITIALISE_ENCLAVE,
+    START_ENCLAVE,
+    STOP_ENCLAVE,
+    GET_ENCLAVE_INSTANCE_INFO_QUOTE,
+    GET_KDS_PERSISTENCE_KEY_SPEC,
+    SET_KDS_PERSISTENCE_KEY,
+    CALL_MESSAGE_HANDLER;
```

Comment:
This one seems quiet generic.

Reply:
So this is for the message handler, which has been left as is (more or less). The messages for the message handler are sometimes commands, sometimes responses, so the name ended up being kinda generic. Disentangling the message handlers individual functions was taking me too long to figure out, so I decided I'd leave it alone for now and raise a separate ticket to deal with it later.